src/pages/AuthenticationPage/AuthenticationPage.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+import "./AuthenticationPage.css";
+import AuthenticationForm from "../../components/AuthenticationForm/AuthenticationForm";
+import AuthenticationLayout from "../../layouts/AuthenticationLayout/AuthenticationLayout";
```

Comment:
```suggestion
import AuthenticationForm from "../../components/AuthenticationForm";
import AuthenticationLayout from "../../layouts/AuthenticationLayout";
```
import statement should never look like AuthenticationForm/AuthenticationForm , same name repeating twice. name file as index.jsx in folder so you can avoid it.

Reply:
I fixed it!